main.py

```
@@ -370,6 +394,48 @@ def _cleanup(done_task: asyncio.Task) -> None:
 
         task.add_done_callback(_cleanup)
 
+    async def _extract_profile_async(self, user_id: str, message: str) -> None:
+        """后台异步提取用户画像标签，写入 raw/user_profile/{user_id}/"""
+        import time
+        try:
+            # 频率控制：同一用户 5 分钟内最多提取一次
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)